src/components/Home/FeedType.tsx

```diff
@@ -1,37 +1,46 @@
 import TabButton from '@components/UI/TabButton';
 import { SparklesIcon, ViewListIcon } from '@heroicons/react/outline';
+import isFeatureEnabled from '@lib/isFeatureEnabled';
 import { Leafwatch } from '@lib/leafwatch';
 import type { Dispatch, FC } from 'react';
+import { useAppStore } from 'src/store/app';
 import { MISCELLANEOUS } from 'src/tracking';
 
+import FeedEventFilters from './FeedEventFilters';
+
 interface Props {
   setFeedType: Dispatch<'TIMELINE' | 'HIGHLIGHTS'>;
   feedType: 'TIMELINE' | 'HIGHLIGHTS';
```

Comment:
**suggestion:** better move to lenstertypes.ts as enum so we can prevent using magic string inside jsx also using magic string is very bad 🙏🏼

Reply:
dei, idhu un code than